packages/frontend/src/components/BlueButton/BlueButton.tsx

```diff
@@ -1,14 +1,8 @@
 import { Button } from '@mantine/core'
+import type { ButtonProps } from '@mantine/core'
 import useStyles from './style'
 
-type BlueButtonProps = {
-  children?: string
-  onClick: (e?: React.MouseEvent<HTMLElement>) => void
-  rightIcon?: React.ReactNode
-  leftIcon?: React.ReactNode
-}
-
-const BlueButton = ({ children, onClick, rightIcon, leftIcon }: BlueButtonProps) => {
+const BlueButton = ({ children, onClick, rightIcon, leftIcon }: ButtonProps<'button'>) => {
```

Comment:
Nope, why you would say someone, that he can pass here all of this stuff, just to discover that you are using only 4 of them? That's nasty lie  😄